identity/signer.go

```diff
@@ -0,0 +1,44 @@
+package identity
+
+import (
+	"github.com/ethereum/go-ethereum/crypto"
+	"fmt"
+)
+
+type Signer interface {
+	Sign(message []byte) ([]byte, error)
+}
+
+type keystoreSigner struct {
+	keystoreManager keystoreInterface
```

Comment:
If it satisfies `keystoreInterface` interface (and not `keystoreManagerInterface`), maybe `Manager` in `keystoreManager` variable name is redundant?

Reply:
Yeah `keystore` is enough

Reply:
Fixed now.